tests/FoodStorage/test_foodStorage.py

```diff
@@ -13,28 +13,52 @@ def setUp(self):
 
     def testInitializeFoodStorage(self):
         self.assertFalse(self.foodStorage.isOpen)
-        self.assertEquals(dateToday, self.foodStorage.current_date)
+        self.assertEqual(dateToday, self.foodStorage.current_date)
         self.assertTrue(isEmptyInventory(self.foodStorage))
+        self.assertEqual(7, self.foodStorage.expiration_window.days)
+
+        self.foodStorage = FoodStorage(expiration_window=8)
+        self.assertFalse(self.foodStorage.isOpen)
+        self.assertEqual(dateToday, self.foodStorage.current_date)
+        self.assertTrue(isEmptyInventory(self.foodStorage))
+        self.assertEqual(8, self.foodStorage.expiration_window.days)
 
     def testAddFoods(self):
         # Inventory should start as empty
         self.assertTrue(isEmptyInventory(self.foodStorage))
 
         # Inventory should be sorted after adding multiple foods
-        addMultipleFoods(self.foodStorage, [food1, food2, food3])
```

Comment:
Why are you removing `addMultipleFoods`?

Reply:
I realized it was kind of unnecessary when we can just call `.addFoods`. It might be unnecessary to have a helper to check for an empty Inventory also, so I can also replace those.